[2015-02-17 12:23:38] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /dev.php"" at C:\xampp\htdocs\vtnwebapp\vendor\symfony\symfony\src\Symfony\Component\HttpKernel\EventListener\RouterListener.php line 145 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /dev.php\" at C:\\xampp\\htdocs\\vtnwebapp\\vendor\\symfony\\symfony\\src\\Symfony\\Component\\HttpKernel\\EventListener\\RouterListener.php:145, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at C:\\xampp\\htdocs\\vtnwebapp\\app\\cache\\prod\\appProdUrlMatcher.php:35)"} []
[2015-02-17 12:23:43] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /"" at C:\xampp\htdocs\vtnwebapp\vendor\symfony\symfony\src\Symfony\Component\HttpKernel\EventListener\RouterListener.php line 145 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /\" at C:\\xampp\\htdocs\\vtnwebapp\\vendor\\symfony\\symfony\\src\\Symfony\\Component\\HttpKernel\\EventListener\\RouterListener.php:145, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at C:\\xampp\\htdocs\\vtnwebapp\\app\\cache\\prod\\appProdUrlMatcher.php:35)"} []
[2015-02-17 12:24:03] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /bundles/framework/images/input_bg.gif" (from "http://localhost/vtnwebapp/web/bundles/sensiodistribution/webconfigurator/css/configurator.css")" at C:\xampp\htdocs\vtnwebapp\app\cache\prod\classes.php line 2420 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /bundles/framework/images/input_bg.gif\" (from \"http://localhost/vtnwebapp/web/bundles/sensiodistribution/webconfigurator/css/configurator.css\") at C:\\xampp\\htdocs\\vtnwebapp\\app\\cache\\prod\\classes.php:2420, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at C:\\xampp\\htdocs\\vtnwebapp\\app\\cache\\prod\\appProdUrlMatcher.php:35)"} []
[2015-02-17 12:33:47] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /bundles/framework/images/input_bg.gif" (from "http://localhost/vtnwebapp/web/bundles/sensiodistribution/webconfigurator/css/configurator.css")" at C:\xampp\htdocs\vtnwebapp\app\cache\prod\classes.php line 2420 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /bundles/framework/images/input_bg.gif\" (from \"http://localhost/vtnwebapp/web/bundles/sensiodistribution/webconfigurator/css/configurator.css\") at C:\\xampp\\htdocs\\vtnwebapp\\app\\cache\\prod\\classes.php:2420, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at C:\\xampp\\htdocs\\vtnwebapp\\app\\cache\\prod\\appProdUrlMatcher.php:35)"} []
[2015-02-17 14:35:39] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /random/100"" at C:\xampp\htdocs\vtnwebapp\app\cache\prod\classes.php line 2420 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /random/100\" at C:\\xampp\\htdocs\\vtnwebapp\\app\\cache\\prod\\classes.php:2420, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at C:\\xampp\\htdocs\\vtnwebapp\\app\\cache\\prod\\appProdUrlMatcher.php:35)"} []
[2015-02-17 14:41:51] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /"" at C:\xampp\htdocs\vtnwebapp\app\cache\prod\classes.php line 2420 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /\" at C:\\xampp\\htdocs\\vtnwebapp\\app\\cache\\prod\\classes.php:2420, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at C:\\xampp\\htdocs\\vtnwebapp\\app\\cache\\prod\\appProdUrlMatcher.php:35)"} []
[2015-02-17 14:56:46] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /" (from "http://localhost/vtnwebapp/")" at C:\xampp\htdocs\vtnwebapp\app\cache\prod\classes.php line 2420 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /\" (from \"http://localhost/vtnwebapp/\") at C:\\xampp\\htdocs\\vtnwebapp\\app\\cache\\prod\\classes.php:2420, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at C:\\xampp\\htdocs\\vtnwebapp\\app\\cache\\prod\\appProdUrlMatcher.php:35)"} []
[2015-02-17 14:56:51] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /"" at C:\xampp\htdocs\vtnwebapp\app\cache\prod\classes.php line 2420 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /\" at C:\\xampp\\htdocs\\vtnwebapp\\app\\cache\\prod\\classes.php:2420, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at C:\\xampp\\htdocs\\vtnwebapp\\app\\cache\\prod\\appProdUrlMatcher.php:35)"} []
[2015-02-17 14:56:57] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /"" at C:\xampp\htdocs\vtnwebapp\app\cache\prod\classes.php line 2420 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /\" at C:\\xampp\\htdocs\\vtnwebapp\\app\\cache\\prod\\classes.php:2420, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at C:\\xampp\\htdocs\\vtnwebapp\\app\\cache\\prod\\appProdUrlMatcher.php:35)"} []
[2015-02-17 15:20:22] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /"" at C:\xampp\htdocs\vtnwebapp\vendor\symfony\symfony\src\Symfony\Component\HttpKernel\EventListener\RouterListener.php line 145 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /\" at C:\\xampp\\htdocs\\vtnwebapp\\vendor\\symfony\\symfony\\src\\Symfony\\Component\\HttpKernel\\EventListener\\RouterListener.php:145, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at C:\\xampp\\htdocs\\vtnwebapp\\app\\cache\\prod\\appProdUrlMatcher.php:35)"} []
[2015-02-17 15:20:22] event.DEBUG: Notified event "kernel.request" to listener "Symfony\Component\EventDispatcher\Debug\WrappedListener::__invoke". [] []
[2015-02-17 15:20:22] event.DEBUG: Notified event "kernel.request" to listener "Symfony\Component\EventDispatcher\Debug\WrappedListener::__invoke". [] []
[2015-02-17 15:20:22] event.DEBUG: Notified event "kernel.request" to listener "Symfony\Component\EventDispatcher\Debug\WrappedListener::__invoke". [] []
[2015-02-17 15:20:22] event.DEBUG: Notified event "kernel.request" to listener "Symfony\Component\EventDispatcher\Debug\WrappedListener::__invoke". [] []
[2015-02-17 15:20:22] event.DEBUG: Notified event "kernel.request" to listener "Symfony\Component\EventDispatcher\Debug\WrappedListener::__invoke". [] []
[2015-02-17 15:20:22] event.DEBUG: Notified event "kernel.request" to listener "Symfony\Component\EventDispatcher\Debug\WrappedListener::__invoke". [] []
[2015-02-17 15:20:22] event.DEBUG: Notified event "kernel.request" to listener "Symfony\Component\EventDispatcher\Debug\WrappedListener::__invoke". [] []
[2015-02-17 15:20:22] event.DEBUG: Notified event "kernel.controller" to listener "Sensio\Bundle\FrameworkExtraBundle\EventListener\ControllerListener::onKernelController". [] []
[2015-02-17 15:20:22] event.DEBUG: Notified event "kernel.controller" to listener "Sensio\Bundle\FrameworkExtraBundle\EventListener\ParamConverterListener::onKernelController". [] []
[2015-02-17 15:20:22] event.DEBUG: Notified event "kernel.controller" to listener "Sensio\Bundle\FrameworkExtraBundle\EventListener\HttpCacheListener::onKernelController". [] []
[2015-02-17 15:20:22] event.DEBUG: Notified event "kernel.controller" to listener "Sensio\Bundle\FrameworkExtraBundle\EventListener\SecurityListener::onKernelController". [] []
[2015-02-17 15:20:22] event.DEBUG: Notified event "kernel.controller" to listener "Sensio\Bundle\FrameworkExtraBundle\EventListener\TemplateListener::onKernelController". [] []
[2015-02-17 15:20:23] event.DEBUG: Notified event "kernel.response" to listener "Symfony\Component\HttpKernel\EventListener\ResponseListener::onKernelResponse". [] []
[2015-02-17 15:20:23] event.DEBUG: Notified event "kernel.response" to listener "Symfony\Component\Security\Http\RememberMe\ResponseListener::onKernelResponse". [] []
[2015-02-17 15:20:23] event.DEBUG: Notified event "kernel.response" to listener "Sensio\Bundle\FrameworkExtraBundle\EventListener\HttpCacheListener::onKernelResponse". [] []
[2015-02-17 15:20:23] event.DEBUG: Notified event "kernel.response" to listener "Symfony\Component\HttpKernel\EventListener\SaveSessionListener::onKernelResponse". [] []
[2015-02-17 15:20:23] event.DEBUG: Notified event "kernel.response" to listener "Symfony\Component\HttpKernel\EventListener\StreamedResponseListener::onKernelResponse". [] []
[2015-02-17 15:20:23] event.DEBUG: Notified event "kernel.finish_request" to listener "Symfony\Component\HttpKernel\EventListener\LocaleListener::onKernelFinishRequest". [] []
[2015-02-17 15:20:23] event.DEBUG: Notified event "kernel.finish_request" to listener "Symfony\Component\HttpKernel\EventListener\TranslatorListener::onKernelFinishRequest". [] []
[2015-02-17 15:20:23] event.DEBUG: Notified event "kernel.finish_request" to listener "Symfony\Component\HttpKernel\EventListener\RouterListener::onKernelFinishRequest". [] []
[2015-02-17 15:20:23] event.DEBUG: Notified event "kernel.finish_request" to listener "Symfony\Component\Security\Http\Firewall::onKernelFinishRequest". [] []
[2015-02-17 15:20:23] event.DEBUG: Notified event "kernel.exception" to listener "Symfony\Component\HttpKernel\EventListener\ExceptionListener::onKernelException". [] []
[2015-02-17 15:20:23] event.DEBUG: Listener "Symfony\Component\HttpKernel\EventListener\ExceptionListener::onKernelException" stopped propagation of the event "kernel.exception". [] []
[2015-02-17 15:20:23] event.DEBUG: Notified event "kernel.response" to listener "Symfony\Component\HttpKernel\EventListener\ResponseListener::onKernelResponse". [] []
[2015-02-17 15:20:23] event.DEBUG: Notified event "kernel.response" to listener "Symfony\Component\Security\Http\RememberMe\ResponseListener::onKernelResponse". [] []
[2015-02-17 15:20:23] event.DEBUG: Notified event "kernel.response" to listener "Sensio\Bundle\FrameworkExtraBundle\EventListener\HttpCacheListener::onKernelResponse". [] []
[2015-02-17 15:20:23] event.DEBUG: Notified event "kernel.response" to listener "Symfony\Component\HttpKernel\EventListener\SaveSessionListener::onKernelResponse". [] []
[2015-02-17 15:20:23] event.DEBUG: Notified event "kernel.response" to listener "Symfony\Component\HttpKernel\EventListener\StreamedResponseListener::onKernelResponse". [] []
[2015-02-17 15:20:23] event.DEBUG: Notified event "kernel.finish_request" to listener "Symfony\Component\HttpKernel\EventListener\LocaleListener::onKernelFinishRequest". [] []
[2015-02-17 15:20:23] event.DEBUG: Notified event "kernel.finish_request" to listener "Symfony\Component\HttpKernel\EventListener\TranslatorListener::onKernelFinishRequest". [] []
[2015-02-17 15:20:23] event.DEBUG: Notified event "kernel.finish_request" to listener "Symfony\Component\HttpKernel\EventListener\RouterListener::onKernelFinishRequest". [] []
[2015-02-17 15:20:23] event.DEBUG: Notified event "kernel.finish_request" to listener "Symfony\Component\Security\Http\Firewall::onKernelFinishRequest". [] []
[2015-02-17 15:20:23] event.DEBUG: Notified event "kernel.terminate" to listener "Symfony\Bundle\SwiftmailerBundle\EventListener\EmailSenderListener::onTerminate". [] []
[2015-02-17 15:20:33] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /"" at C:\xampp\htdocs\vtnwebapp\app\cache\prod\classes.php line 2420 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /\" at C:\\xampp\\htdocs\\vtnwebapp\\app\\cache\\prod\\classes.php:2420, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at C:\\xampp\\htdocs\\vtnwebapp\\app\\cache\\prod\\appProdUrlMatcher.php:35)"} []
[2015-02-17 15:20:33] event.DEBUG: Notified event "kernel.request" to listener "Symfony\Component\EventDispatcher\Debug\WrappedListener::__invoke". [] []
[2015-02-17 15:20:33] event.DEBUG: Notified event "kernel.request" to listener "Symfony\Component\EventDispatcher\Debug\WrappedListener::__invoke". [] []
[2015-02-17 15:20:33] event.DEBUG: Notified event "kernel.request" to listener "Symfony\Component\EventDispatcher\Debug\WrappedListener::__invoke". [] []
[2015-02-17 15:20:33] event.DEBUG: Notified event "kernel.request" to listener "Symfony\Component\EventDispatcher\Debug\WrappedListener::__invoke". [] []
[2015-02-17 15:20:33] event.DEBUG: Notified event "kernel.request" to listener "Symfony\Component\EventDispatcher\Debug\WrappedListener::__invoke". [] []
[2015-02-17 15:20:33] event.DEBUG: Notified event "kernel.request" to listener "Symfony\Component\EventDispatcher\Debug\WrappedListener::__invoke". [] []
[2015-02-17 15:20:33] event.DEBUG: Notified event "kernel.request" to listener "Symfony\Component\EventDispatcher\Debug\WrappedListener::__invoke". [] []
[2015-02-17 15:20:33] event.DEBUG: Notified event "kernel.controller" to listener "Sensio\Bundle\FrameworkExtraBundle\EventListener\ControllerListener::onKernelController". [] []
[2015-02-17 15:20:33] event.DEBUG: Notified event "kernel.controller" to listener "Sensio\Bundle\FrameworkExtraBundle\EventListener\ParamConverterListener::onKernelController". [] []
[2015-02-17 15:20:33] event.DEBUG: Notified event "kernel.controller" to listener "Sensio\Bundle\FrameworkExtraBundle\EventListener\HttpCacheListener::onKernelController". [] []
[2015-02-17 15:20:33] event.DEBUG: Notified event "kernel.controller" to listener "Sensio\Bundle\FrameworkExtraBundle\EventListener\SecurityListener::onKernelController". [] []
[2015-02-17 15:20:33] event.DEBUG: Notified event "kernel.controller" to listener "Sensio\Bundle\FrameworkExtraBundle\EventListener\TemplateListener::onKernelController". [] []
[2015-02-17 15:20:34] event.DEBUG: Notified event "kernel.response" to listener "Symfony\Component\HttpKernel\EventListener\ResponseListener::onKernelResponse". [] []
[2015-02-17 15:20:34] event.DEBUG: Notified event "kernel.response" to listener "Symfony\Component\Security\Http\RememberMe\ResponseListener::onKernelResponse". [] []
[2015-02-17 15:20:34] event.DEBUG: Notified event "kernel.response" to listener "Sensio\Bundle\FrameworkExtraBundle\EventListener\HttpCacheListener::onKernelResponse". [] []
[2015-02-17 15:20:34] event.DEBUG: Notified event "kernel.response" to listener "Symfony\Component\HttpKernel\EventListener\SaveSessionListener::onKernelResponse". [] []
[2015-02-17 15:20:34] event.DEBUG: Notified event "kernel.response" to listener "Symfony\Component\HttpKernel\EventListener\StreamedResponseListener::onKernelResponse". [] []
[2015-02-17 15:20:34] event.DEBUG: Notified event "kernel.finish_request" to listener "Symfony\Component\HttpKernel\EventListener\LocaleListener::onKernelFinishRequest". [] []
[2015-02-17 15:20:34] event.DEBUG: Notified event "kernel.finish_request" to listener "Symfony\Component\HttpKernel\EventListener\TranslatorListener::onKernelFinishRequest". [] []
[2015-02-17 15:20:34] event.DEBUG: Notified event "kernel.finish_request" to listener "Symfony\Component\HttpKernel\EventListener\RouterListener::onKernelFinishRequest". [] []
[2015-02-17 15:20:34] event.DEBUG: Notified event "kernel.finish_request" to listener "Symfony\Component\Security\Http\Firewall::onKernelFinishRequest". [] []
[2015-02-17 15:20:34] event.DEBUG: Notified event "kernel.exception" to listener "Symfony\Component\HttpKernel\EventListener\ExceptionListener::onKernelException". [] []
[2015-02-17 15:20:34] event.DEBUG: Listener "Symfony\Component\HttpKernel\EventListener\ExceptionListener::onKernelException" stopped propagation of the event "kernel.exception". [] []
[2015-02-17 15:20:34] event.DEBUG: Notified event "kernel.response" to listener "Symfony\Component\HttpKernel\EventListener\ResponseListener::onKernelResponse". [] []
[2015-02-17 15:20:34] event.DEBUG: Notified event "kernel.response" to listener "Symfony\Component\Security\Http\RememberMe\ResponseListener::onKernelResponse". [] []
[2015-02-17 15:20:34] event.DEBUG: Notified event "kernel.response" to listener "Sensio\Bundle\FrameworkExtraBundle\EventListener\HttpCacheListener::onKernelResponse". [] []
[2015-02-17 15:20:34] event.DEBUG: Notified event "kernel.response" to listener "Symfony\Component\HttpKernel\EventListener\SaveSessionListener::onKernelResponse". [] []
[2015-02-17 15:20:34] event.DEBUG: Notified event "kernel.response" to listener "Symfony\Component\HttpKernel\EventListener\StreamedResponseListener::onKernelResponse". [] []
[2015-02-17 15:20:34] event.DEBUG: Notified event "kernel.finish_request" to listener "Symfony\Component\HttpKernel\EventListener\LocaleListener::onKernelFinishRequest". [] []
[2015-02-17 15:20:34] event.DEBUG: Notified event "kernel.finish_request" to listener "Symfony\Component\HttpKernel\EventListener\TranslatorListener::onKernelFinishRequest". [] []
[2015-02-17 15:20:34] event.DEBUG: Notified event "kernel.finish_request" to listener "Symfony\Component\HttpKernel\EventListener\RouterListener::onKernelFinishRequest". [] []
[2015-02-17 15:20:34] event.DEBUG: Notified event "kernel.finish_request" to listener "Symfony\Component\Security\Http\Firewall::onKernelFinishRequest". [] []
[2015-02-17 15:20:34] event.DEBUG: Notified event "kernel.terminate" to listener "Symfony\Bundle\SwiftmailerBundle\EventListener\EmailSenderListener::onTerminate". [] []
[2015-02-17 15:20:35] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /random/10"" at C:\xampp\htdocs\vtnwebapp\app\cache\prod\classes.php line 2420 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /random/10\" at C:\\xampp\\htdocs\\vtnwebapp\\app\\cache\\prod\\classes.php:2420, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at C:\\xampp\\htdocs\\vtnwebapp\\app\\cache\\prod\\appProdUrlMatcher.php:35)"} []
[2015-02-17 15:20:35] event.DEBUG: Notified event "kernel.request" to listener "Symfony\Component\EventDispatcher\Debug\WrappedListener::__invoke". [] []
[2015-02-17 15:20:35] event.DEBUG: Notified event "kernel.request" to listener "Symfony\Component\EventDispatcher\Debug\WrappedListener::__invoke". [] []
[2015-02-17 15:20:35] event.DEBUG: Notified event "kernel.request" to listener "Symfony\Component\EventDispatcher\Debug\WrappedListener::__invoke". [] []
[2015-02-17 15:20:35] event.DEBUG: Notified event "kernel.request" to listener "Symfony\Component\EventDispatcher\Debug\WrappedListener::__invoke". [] []
[2015-02-17 15:20:35] event.DEBUG: Notified event "kernel.request" to listener "Symfony\Component\EventDispatcher\Debug\WrappedListener::__invoke". [] []
[2015-02-17 15:20:35] event.DEBUG: Notified event "kernel.request" to listener "Symfony\Component\EventDispatcher\Debug\WrappedListener::__invoke". [] []
[2015-02-17 15:20:35] event.DEBUG: Notified event "kernel.request" to listener "Symfony\Component\EventDispatcher\Debug\WrappedListener::__invoke". [] []
[2015-02-17 15:20:35] event.DEBUG: Notified event "kernel.controller" to listener "Sensio\Bundle\FrameworkExtraBundle\EventListener\ControllerListener::onKernelController". [] []
[2015-02-17 15:20:35] event.DEBUG: Notified event "kernel.controller" to listener "Sensio\Bundle\FrameworkExtraBundle\EventListener\ParamConverterListener::onKernelController". [] []
[2015-02-17 15:20:35] event.DEBUG: Notified event "kernel.controller" to listener "Sensio\Bundle\FrameworkExtraBundle\EventListener\HttpCacheListener::onKernelController". [] []
[2015-02-17 15:20:35] event.DEBUG: Notified event "kernel.controller" to listener "Sensio\Bundle\FrameworkExtraBundle\EventListener\SecurityListener::onKernelController". [] []
[2015-02-17 15:20:35] event.DEBUG: Notified event "kernel.controller" to listener "Sensio\Bundle\FrameworkExtraBundle\EventListener\TemplateListener::onKernelController". [] []
[2015-02-17 15:20:35] event.DEBUG: Notified event "kernel.response" to listener "Symfony\Component\HttpKernel\EventListener\ResponseListener::onKernelResponse". [] []
[2015-02-17 15:20:35] event.DEBUG: Notified event "kernel.response" to listener "Symfony\Component\Security\Http\RememberMe\ResponseListener::onKernelResponse". [] []
[2015-02-17 15:20:35] event.DEBUG: Notified event "kernel.response" to listener "Sensio\Bundle\FrameworkExtraBundle\EventListener\HttpCacheListener::onKernelResponse". [] []
[2015-02-17 15:20:35] event.DEBUG: Notified event "kernel.response" to listener "Symfony\Component\HttpKernel\EventListener\SaveSessionListener::onKernelResponse". [] []
[2015-02-17 15:20:35] event.DEBUG: Notified event "kernel.response" to listener "Symfony\Component\HttpKernel\EventListener\StreamedResponseListener::onKernelResponse". [] []
[2015-02-17 15:20:35] event.DEBUG: Notified event "kernel.finish_request" to listener "Symfony\Component\HttpKernel\EventListener\LocaleListener::onKernelFinishRequest". [] []
[2015-02-17 15:20:35] event.DEBUG: Notified event "kernel.finish_request" to listener "Symfony\Component\HttpKernel\EventListener\TranslatorListener::onKernelFinishRequest". [] []
[2015-02-17 15:20:35] event.DEBUG: Notified event "kernel.finish_request" to listener "Symfony\Component\HttpKernel\EventListener\RouterListener::onKernelFinishRequest". [] []
[2015-02-17 15:20:35] event.DEBUG: Notified event "kernel.finish_request" to listener "Symfony\Component\Security\Http\Firewall::onKernelFinishRequest". [] []
[2015-02-17 15:20:35] event.DEBUG: Notified event "kernel.exception" to listener "Symfony\Component\HttpKernel\EventListener\ExceptionListener::onKernelException". [] []
[2015-02-17 15:20:35] event.DEBUG: Listener "Symfony\Component\HttpKernel\EventListener\ExceptionListener::onKernelException" stopped propagation of the event "kernel.exception". [] []
[2015-02-17 15:20:35] event.DEBUG: Notified event "kernel.response" to listener "Symfony\Component\HttpKernel\EventListener\ResponseListener::onKernelResponse". [] []
[2015-02-17 15:20:35] event.DEBUG: Notified event "kernel.response" to listener "Symfony\Component\Security\Http\RememberMe\ResponseListener::onKernelResponse". [] []
[2015-02-17 15:20:35] event.DEBUG: Notified event "kernel.response" to listener "Sensio\Bundle\FrameworkExtraBundle\EventListener\HttpCacheListener::onKernelResponse". [] []
[2015-02-17 15:20:35] event.DEBUG: Notified event "kernel.response" to listener "Symfony\Component\HttpKernel\EventListener\SaveSessionListener::onKernelResponse". [] []
[2015-02-17 15:20:35] event.DEBUG: Notified event "kernel.response" to listener "Symfony\Component\HttpKernel\EventListener\StreamedResponseListener::onKernelResponse". [] []
[2015-02-17 15:20:35] event.DEBUG: Notified event "kernel.finish_request" to listener "Symfony\Component\HttpKernel\EventListener\LocaleListener::onKernelFinishRequest". [] []
[2015-02-17 15:20:35] event.DEBUG: Notified event "kernel.finish_request" to listener "Symfony\Component\HttpKernel\EventListener\TranslatorListener::onKernelFinishRequest". [] []
[2015-02-17 15:20:35] event.DEBUG: Notified event "kernel.finish_request" to listener "Symfony\Component\HttpKernel\EventListener\RouterListener::onKernelFinishRequest". [] []
[2015-02-17 15:20:35] event.DEBUG: Notified event "kernel.finish_request" to listener "Symfony\Component\Security\Http\Firewall::onKernelFinishRequest". [] []
[2015-02-17 15:20:35] event.DEBUG: Notified event "kernel.terminate" to listener "Symfony\Bundle\SwiftmailerBundle\EventListener\EmailSenderListener::onTerminate". [] []
[2015-02-17 15:21:39] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Config\Exception\FileLoaderLoadException: "Bundle "AcmeDemoBundle" does not exist or it is not enabled. Maybe you forgot to add it in the registerBundles() method of your AppKernel.php file? in @AcmeDemoBundle/Resources/config/routing.yml (which is being imported from "C:\xampp\htdocs\vtnwebapp\app/config/routing.yml"). Make sure the "AcmeDemoBundle/Resources/config/routing.yml" bundle is correctly registered and loaded in the application kernel class." at C:\xampp\htdocs\vtnwebapp\vendor\symfony\symfony\src\Symfony\Component\Config\Loader\FileLoader.php line 120 {"exception":"[object] (Symfony\\Component\\Config\\Exception\\FileLoaderLoadException(code: 0): Bundle \"AcmeDemoBundle\" does not exist or it is not enabled. Maybe you forgot to add it in the registerBundles() method of your AppKernel.php file? in @AcmeDemoBundle/Resources/config/routing.yml (which is being imported from \"C:\\xampp\\htdocs\\vtnwebapp\\app/config/routing.yml\"). Make sure the \"AcmeDemoBundle/Resources/config/routing.yml\" bundle is correctly registered and loaded in the application kernel class. at C:\\xampp\\htdocs\\vtnwebapp\\vendor\\symfony\\symfony\\src\\Symfony\\Component\\Config\\Loader\\FileLoader.php:120, InvalidArgumentException(code: 0): Bundle \"AcmeDemoBundle\" does not exist or it is not enabled. Maybe you forgot to add it in the registerBundles() method of your AppKernel.php file? at C:\\xampp\\htdocs\\vtnwebapp\\app\\bootstrap.php.cache:2399)"} []
[2015-02-17 15:21:39] event.DEBUG: Notified event "kernel.request" to listener "Symfony\Component\EventDispatcher\Debug\WrappedListener::__invoke". [] []
[2015-02-17 15:21:39] event.DEBUG: Notified event "kernel.request" to listener "Symfony\Component\EventDispatcher\Debug\WrappedListener::__invoke". [] []
[2015-02-17 15:21:39] event.DEBUG: Notified event "kernel.request" to listener "Symfony\Component\EventDispatcher\Debug\WrappedListener::__invoke". [] []
[2015-02-17 15:21:39] event.DEBUG: Notified event "kernel.request" to listener "Symfony\Component\EventDispatcher\Debug\WrappedListener::__invoke". [] []
[2015-02-17 15:21:39] event.DEBUG: Notified event "kernel.request" to listener "Symfony\Component\EventDispatcher\Debug\WrappedListener::__invoke". [] []
[2015-02-17 15:21:39] event.DEBUG: Notified event "kernel.request" to listener "Symfony\Component\EventDispatcher\Debug\WrappedListener::__invoke". [] []
[2015-02-17 15:21:39] event.DEBUG: Notified event "kernel.request" to listener "Symfony\Component\EventDispatcher\Debug\WrappedListener::__invoke". [] []
[2015-02-17 15:21:39] event.DEBUG: Notified event "kernel.controller" to listener "Sensio\Bundle\FrameworkExtraBundle\EventListener\ControllerListener::onKernelController". [] []
[2015-02-17 15:21:39] event.DEBUG: Notified event "kernel.controller" to listener "Sensio\Bundle\FrameworkExtraBundle\EventListener\ParamConverterListener::onKernelController". [] []
[2015-02-17 15:21:39] event.DEBUG: Notified event "kernel.controller" to listener "Sensio\Bundle\FrameworkExtraBundle\EventListener\HttpCacheListener::onKernelController". [] []
[2015-02-17 15:21:39] event.DEBUG: Notified event "kernel.controller" to listener "Sensio\Bundle\FrameworkExtraBundle\EventListener\SecurityListener::onKernelController". [] []
[2015-02-17 15:21:39] event.DEBUG: Notified event "kernel.controller" to listener "Sensio\Bundle\FrameworkExtraBundle\EventListener\TemplateListener::onKernelController". [] []
[2015-02-17 15:21:40] event.DEBUG: Notified event "kernel.response" to listener "Symfony\Component\HttpKernel\EventListener\ResponseListener::onKernelResponse". [] []
[2015-02-17 15:21:40] event.DEBUG: Notified event "kernel.response" to listener "Symfony\Component\Security\Http\RememberMe\ResponseListener::onKernelResponse". [] []
[2015-02-17 15:21:40] event.DEBUG: Notified event "kernel.response" to listener "Sensio\Bundle\FrameworkExtraBundle\EventListener\HttpCacheListener::onKernelResponse". [] []
[2015-02-17 15:21:40] event.DEBUG: Notified event "kernel.response" to listener "Symfony\Component\HttpKernel\EventListener\SaveSessionListener::onKernelResponse". [] []
[2015-02-17 15:21:40] event.DEBUG: Notified event "kernel.response" to listener "Symfony\Component\HttpKernel\EventListener\StreamedResponseListener::onKernelResponse". [] []
[2015-02-17 15:21:40] event.DEBUG: Notified event "kernel.finish_request" to listener "Symfony\Component\HttpKernel\EventListener\LocaleListener::onKernelFinishRequest". [] []
[2015-02-17 15:21:40] event.DEBUG: Notified event "kernel.finish_request" to listener "Symfony\Component\HttpKernel\EventListener\TranslatorListener::onKernelFinishRequest". [] []
[2015-02-17 15:21:40] event.DEBUG: Notified event "kernel.finish_request" to listener "Symfony\Component\HttpKernel\EventListener\RouterListener::onKernelFinishRequest". [] []
[2015-02-17 15:21:40] event.DEBUG: Notified event "kernel.finish_request" to listener "Symfony\Component\Security\Http\Firewall::onKernelFinishRequest". [] []
[2015-02-17 15:21:40] event.DEBUG: Notified event "kernel.exception" to listener "Symfony\Component\HttpKernel\EventListener\ExceptionListener::onKernelException". [] []
[2015-02-17 15:21:40] event.DEBUG: Listener "Symfony\Component\HttpKernel\EventListener\ExceptionListener::onKernelException" stopped propagation of the event "kernel.exception". [] []
[2015-02-17 15:21:40] event.DEBUG: Notified event "kernel.response" to listener "Symfony\Component\HttpKernel\EventListener\ResponseListener::onKernelResponse". [] []
[2015-02-17 15:21:40] event.DEBUG: Notified event "kernel.response" to listener "Symfony\Component\Security\Http\RememberMe\ResponseListener::onKernelResponse". [] []
[2015-02-17 15:21:40] event.DEBUG: Notified event "kernel.response" to listener "Sensio\Bundle\FrameworkExtraBundle\EventListener\HttpCacheListener::onKernelResponse". [] []
[2015-02-17 15:21:40] event.DEBUG: Notified event "kernel.response" to listener "Symfony\Component\HttpKernel\EventListener\SaveSessionListener::onKernelResponse". [] []
[2015-02-17 15:21:40] event.DEBUG: Notified event "kernel.response" to listener "Symfony\Component\HttpKernel\EventListener\StreamedResponseListener::onKernelResponse". [] []
[2015-02-17 15:21:40] event.DEBUG: Notified event "kernel.finish_request" to listener "Symfony\Component\HttpKernel\EventListener\LocaleListener::onKernelFinishRequest". [] []
[2015-02-17 15:21:40] event.DEBUG: Notified event "kernel.finish_request" to listener "Symfony\Component\HttpKernel\EventListener\TranslatorListener::onKernelFinishRequest". [] []
[2015-02-17 15:21:40] event.DEBUG: Notified event "kernel.finish_request" to listener "Symfony\Component\HttpKernel\EventListener\RouterListener::onKernelFinishRequest". [] []
[2015-02-17 15:21:40] event.DEBUG: Notified event "kernel.finish_request" to listener "Symfony\Component\Security\Http\Firewall::onKernelFinishRequest". [] []
[2015-02-17 15:21:40] event.DEBUG: Notified event "kernel.terminate" to listener "Symfony\Bundle\SwiftmailerBundle\EventListener\EmailSenderListener::onTerminate". [] []
[2015-02-17 15:24:15] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /"" at C:\xampp\htdocs\vtnwebapp\app\cache\prod\classes.php line 2420 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /\" at C:\\xampp\\htdocs\\vtnwebapp\\app\\cache\\prod\\classes.php:2420, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at C:\\xampp\\htdocs\\vtnwebapp\\app\\cache\\prod\\appProdUrlMatcher.php:35)"} []
[2015-02-17 15:52:58] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /bundles/framework/images/input_bg.gif" (from "http://localhost/vtnwebapp/web/bundles/sensiodistribution/webconfigurator/css/configurator.css")" at C:\xampp\htdocs\vtnwebapp\app\cache\prod\classes.php line 2420 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /bundles/framework/images/input_bg.gif\" (from \"http://localhost/vtnwebapp/web/bundles/sensiodistribution/webconfigurator/css/configurator.css\") at C:\\xampp\\htdocs\\vtnwebapp\\app\\cache\\prod\\classes.php:2420, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at C:\\xampp\\htdocs\\vtnwebapp\\app\\cache\\prod\\appProdUrlMatcher.php:35)"} []
[2015-02-17 15:53:35] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /bundles/framework/images/input_bg.gif" (from "http://localhost/vtnwebapp/web/bundles/sensiodistribution/webconfigurator/css/configurator.css")" at C:\xampp\htdocs\vtnwebapp\app\cache\prod\classes.php line 2420 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /bundles/framework/images/input_bg.gif\" (from \"http://localhost/vtnwebapp/web/bundles/sensiodistribution/webconfigurator/css/configurator.css\") at C:\\xampp\\htdocs\\vtnwebapp\\app\\cache\\prod\\classes.php:2420, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at C:\\xampp\\htdocs\\vtnwebapp\\app\\cache\\prod\\appProdUrlMatcher.php:35)"} []
[2015-02-17 15:53:46] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /bundles/framework/images/input_bg.gif" (from "http://localhost/vtnwebapp/web/bundles/sensiodistribution/webconfigurator/css/configurator.css")" at C:\xampp\htdocs\vtnwebapp\app\cache\prod\classes.php line 2420 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /bundles/framework/images/input_bg.gif\" (from \"http://localhost/vtnwebapp/web/bundles/sensiodistribution/webconfigurator/css/configurator.css\") at C:\\xampp\\htdocs\\vtnwebapp\\app\\cache\\prod\\classes.php:2420, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at C:\\xampp\\htdocs\\vtnwebapp\\app\\cache\\prod\\appProdUrlMatcher.php:35)"} []
[2015-02-17 15:54:07] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /bundles/framework/images/input_bg.gif" (from "http://localhost/vtnwebapp/web/bundles/sensiodistribution/webconfigurator/css/configurator.css")" at C:\xampp\htdocs\vtnwebapp\app\cache\prod\classes.php line 2420 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /bundles/framework/images/input_bg.gif\" (from \"http://localhost/vtnwebapp/web/bundles/sensiodistribution/webconfigurator/css/configurator.css\") at C:\\xampp\\htdocs\\vtnwebapp\\app\\cache\\prod\\classes.php:2420, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at C:\\xampp\\htdocs\\vtnwebapp\\app\\cache\\prod\\appProdUrlMatcher.php:35)"} []
[2015-02-17 15:54:55] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /bundles/framework/images/input_bg.gif" (from "http://localhost/vtnwebapp/web/bundles/sensiodistribution/webconfigurator/css/configurator.css")" at C:\xampp\htdocs\vtnwebapp\app\cache\prod\classes.php line 2420 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /bundles/framework/images/input_bg.gif\" (from \"http://localhost/vtnwebapp/web/bundles/sensiodistribution/webconfigurator/css/configurator.css\") at C:\\xampp\\htdocs\\vtnwebapp\\app\\cache\\prod\\classes.php:2420, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at C:\\xampp\\htdocs\\vtnwebapp\\app\\cache\\prod\\appProdUrlMatcher.php:35)"} []
[2015-02-17 15:57:20] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /"" at C:\xampp\htdocs\vtnwebapp\vendor\symfony\symfony\src\Symfony\Component\HttpKernel\EventListener\RouterListener.php line 145 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /\" at C:\\xampp\\htdocs\\vtnwebapp\\vendor\\symfony\\symfony\\src\\Symfony\\Component\\HttpKernel\\EventListener\\RouterListener.php:145, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at C:\\xampp\\htdocs\\vtnwebapp\\app\\cache\\prod\\appProdUrlMatcher.php:35)"} []
[2015-02-17 15:57:31] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /"" at C:\xampp\htdocs\vtnwebapp\app\cache\prod\classes.php line 2420 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /\" at C:\\xampp\\htdocs\\vtnwebapp\\app\\cache\\prod\\classes.php:2420, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at C:\\xampp\\htdocs\\vtnwebapp\\app\\cache\\prod\\appProdUrlMatcher.php:35)"} []
[2015-02-17 15:57:32] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /random/1000"" at C:\xampp\htdocs\vtnwebapp\app\cache\prod\classes.php line 2420 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /random/1000\" at C:\\xampp\\htdocs\\vtnwebapp\\app\\cache\\prod\\classes.php:2420, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at C:\\xampp\\htdocs\\vtnwebapp\\app\\cache\\prod\\appProdUrlMatcher.php:35)"} []
[2015-02-24 07:17:03] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /"" at C:\xampp\htdocs\vtnwebapp\vendor\symfony\symfony\src\Symfony\Component\HttpKernel\EventListener\RouterListener.php line 145 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /\" at C:\\xampp\\htdocs\\vtnwebapp\\vendor\\symfony\\symfony\\src\\Symfony\\Component\\HttpKernel\\EventListener\\RouterListener.php:145, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at C:\\xampp\\htdocs\\vtnwebapp\\app\\cache\\prod\\appProdUrlMatcher.php:35)"} []
[2015-02-24 07:17:17] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /random/10"" at C:\xampp\htdocs\vtnwebapp\app\cache\prod\classes.php line 2420 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /random/10\" at C:\\xampp\\htdocs\\vtnwebapp\\app\\cache\\prod\\classes.php:2420, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at C:\\xampp\\htdocs\\vtnwebapp\\app\\cache\\prod\\appProdUrlMatcher.php:35)"} []
[2015-02-24 07:18:51] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /"" at C:\xampp\htdocs\vtnwebapp\app\cache\prod\classes.php line 2420 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /\" at C:\\xampp\\htdocs\\vtnwebapp\\app\\cache\\prod\\classes.php:2420, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at C:\\xampp\\htdocs\\vtnwebapp\\app\\cache\\prod\\appProdUrlMatcher.php:35)"} []
[2015-02-24 07:31:19] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /random/10"" at C:\xampp\htdocs\vtnwebapp\app\cache\prod\classes.php line 2420 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /random/10\" at C:\\xampp\\htdocs\\vtnwebapp\\app\\cache\\prod\\classes.php:2420, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at C:\\xampp\\htdocs\\vtnwebapp\\app\\cache\\prod\\appProdUrlMatcher.php:35)"} []
[2015-02-24 07:31:45] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /random/10"" at C:\xampp\htdocs\vtnwebapp\app\cache\prod\classes.php line 2420 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /random/10\" at C:\\xampp\\htdocs\\vtnwebapp\\app\\cache\\prod\\classes.php:2420, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at C:\\xampp\\htdocs\\vtnwebapp\\app\\cache\\prod\\appProdUrlMatcher.php:35)"} []
[2015-02-24 07:31:52] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /random/10"" at C:\xampp\htdocs\vtnwebapp\app\cache\prod\classes.php line 2420 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /random/10\" at C:\\xampp\\htdocs\\vtnwebapp\\app\\cache\\prod\\classes.php:2420, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at C:\\xampp\\htdocs\\vtnwebapp\\app\\cache\\prod\\appProdUrlMatcher.php:35)"} []
[2015-02-24 07:31:55] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /"" at C:\xampp\htdocs\vtnwebapp\app\cache\prod\classes.php line 2420 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /\" at C:\\xampp\\htdocs\\vtnwebapp\\app\\cache\\prod\\classes.php:2420, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at C:\\xampp\\htdocs\\vtnwebapp\\app\\cache\\prod\\appProdUrlMatcher.php:35)"} []
[2015-02-24 07:36:27] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /"" at C:\xampp\htdocs\vtnwebapp\app\cache\prod\classes.php line 2420 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /\" at C:\\xampp\\htdocs\\vtnwebapp\\app\\cache\\prod\\classes.php:2420, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at C:\\xampp\\htdocs\\vtnwebapp\\app\\cache\\prod\\appProdUrlMatcher.php:35)"} []
[2015-02-24 07:36:32] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /random/10"" at C:\xampp\htdocs\vtnwebapp\app\cache\prod\classes.php line 2420 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /random/10\" at C:\\xampp\\htdocs\\vtnwebapp\\app\\cache\\prod\\classes.php:2420, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at C:\\xampp\\htdocs\\vtnwebapp\\app\\cache\\prod\\appProdUrlMatcher.php:35)"} []
[2015-02-24 07:37:11] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /random/10"" at C:\xampp\htdocs\vtnwebapp\app\cache\prod\classes.php line 2420 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /random/10\" at C:\\xampp\\htdocs\\vtnwebapp\\app\\cache\\prod\\classes.php:2420, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at C:\\xampp\\htdocs\\vtnwebapp\\app\\cache\\prod\\appProdUrlMatcher.php:35)"} []
[2015-02-24 07:51:15] request.INFO: Matched route "random" (parameters: "_controller": "AppBundle\Controller\RandomController::indexAction", "limit": "1000", "_route": "random") [] []
[2015-02-24 07:51:16] request.CRITICAL: Uncaught PHP Exception InvalidArgumentException: "Unable to find template "AcmeDemoBundle:Random:index.html.twig"." at C:\xampp\htdocs\vtnwebapp\vendor\symfony\symfony\src\Symfony\Bridge\Twig\TwigEngine.php line 128 {"exception":"[object] (InvalidArgumentException(code: 0): Unable to find template \"AcmeDemoBundle:Random:index.html.twig\". at C:\\xampp\\htdocs\\vtnwebapp\\vendor\\symfony\\symfony\\src\\Symfony\\Bridge\\Twig\\TwigEngine.php:128, Twig_Error_Loader(code: 0): Unable to find template \"AcmeDemoBundle:Random:index.html.twig\". at C:\\xampp\\htdocs\\vtnwebapp\\vendor\\symfony\\symfony\\src\\Symfony\\Bundle\\TwigBundle\\Loader\\FilesystemLoader.php:91, InvalidArgumentException(code: 0): Template name \"AcmeDemoBundle:Random:index.html.twig\" is not valid. at C:\\xampp\\htdocs\\vtnwebapp\\app\\cache\\prod\\classes.php:756, InvalidArgumentException(code: 0): Bundle \"AcmeDemoBundle\" does not exist or it is not enabled. Maybe you forgot to add it in the registerBundles() method of your AppKernel.php file? at C:\\xampp\\htdocs\\vtnwebapp\\app\\bootstrap.php.cache:2399)"} []
[2015-02-24 07:51:18] request.INFO: Matched route "random" (parameters: "_controller": "AppBundle\Controller\RandomController::indexAction", "limit": "1000", "_route": "random") [] []
[2015-02-24 07:51:18] request.CRITICAL: Uncaught PHP Exception InvalidArgumentException: "Unable to find template "AcmeDemoBundle:Random:index.html.twig"." at C:\xampp\htdocs\vtnwebapp\vendor\symfony\symfony\src\Symfony\Bridge\Twig\TwigEngine.php line 128 {"exception":"[object] (InvalidArgumentException(code: 0): Unable to find template \"AcmeDemoBundle:Random:index.html.twig\". at C:\\xampp\\htdocs\\vtnwebapp\\vendor\\symfony\\symfony\\src\\Symfony\\Bridge\\Twig\\TwigEngine.php:128, Twig_Error_Loader(code: 0): Unable to find template \"AcmeDemoBundle:Random:index.html.twig\". at C:\\xampp\\htdocs\\vtnwebapp\\vendor\\symfony\\symfony\\src\\Symfony\\Bundle\\TwigBundle\\Loader\\FilesystemLoader.php:91, InvalidArgumentException(code: 0): Template name \"AcmeDemoBundle:Random:index.html.twig\" is not valid. at C:\\xampp\\htdocs\\vtnwebapp\\app\\cache\\prod\\classes.php:756, InvalidArgumentException(code: 0): Bundle \"AcmeDemoBundle\" does not exist or it is not enabled. Maybe you forgot to add it in the registerBundles() method of your AppKernel.php file? at C:\\xampp\\htdocs\\vtnwebapp\\app\\bootstrap.php.cache:2399)"} []
[2015-02-24 07:53:09] request.INFO: Matched route "random" (parameters: "_controller": "AppBundle\Controller\RandomController::indexAction", "limit": "1000", "_route": "random") [] []
[2015-02-24 07:53:09] request.CRITICAL: Uncaught PHP Exception InvalidArgumentException: "Unable to find template "AppBundle:Random:index.html.twig"." at C:\xampp\htdocs\vtnwebapp\vendor\symfony\symfony\src\Symfony\Bridge\Twig\TwigEngine.php line 128 {"exception":"[object] (InvalidArgumentException(code: 0): Unable to find template \"AppBundle:Random:index.html.twig\". at C:\\xampp\\htdocs\\vtnwebapp\\vendor\\symfony\\symfony\\src\\Symfony\\Bridge\\Twig\\TwigEngine.php:128, Twig_Error_Loader(code: 0): Unable to find template \"AppBundle:Random:index.html.twig\". at C:\\xampp\\htdocs\\vtnwebapp\\vendor\\symfony\\symfony\\src\\Symfony\\Bundle\\TwigBundle\\Loader\\FilesystemLoader.php:91, InvalidArgumentException(code: 0): Unable to find template \"AppBundle:Random:index.html.twig\" : \"Unable to find file \"@AppBundle/Resources/views/Random/index.html.twig\".\". at C:\\xampp\\htdocs\\vtnwebapp\\app\\cache\\prod\\classes.php:801, InvalidArgumentException(code: 0): Unable to find file \"@AppBundle/Resources/views/Random/index.html.twig\". at C:\\xampp\\htdocs\\vtnwebapp\\app\\bootstrap.php.cache:2449)"} []
[2015-02-24 07:53:11] request.INFO: Matched route "random" (parameters: "_controller": "AppBundle\Controller\RandomController::indexAction", "limit": "1000", "_route": "random") [] []
[2015-02-24 07:53:11] request.CRITICAL: Uncaught PHP Exception InvalidArgumentException: "Unable to find template "AppBundle:Random:index.html.twig"." at C:\xampp\htdocs\vtnwebapp\vendor\symfony\symfony\src\Symfony\Bridge\Twig\TwigEngine.php line 128 {"exception":"[object] (InvalidArgumentException(code: 0): Unable to find template \"AppBundle:Random:index.html.twig\". at C:\\xampp\\htdocs\\vtnwebapp\\vendor\\symfony\\symfony\\src\\Symfony\\Bridge\\Twig\\TwigEngine.php:128, Twig_Error_Loader(code: 0): Unable to find template \"AppBundle:Random:index.html.twig\". at C:\\xampp\\htdocs\\vtnwebapp\\vendor\\symfony\\symfony\\src\\Symfony\\Bundle\\TwigBundle\\Loader\\FilesystemLoader.php:91, InvalidArgumentException(code: 0): Unable to find template \"AppBundle:Random:index.html.twig\" : \"Unable to find file \"@AppBundle/Resources/views/Random/index.html.twig\".\". at C:\\xampp\\htdocs\\vtnwebapp\\app\\cache\\prod\\classes.php:801, InvalidArgumentException(code: 0): Unable to find file \"@AppBundle/Resources/views/Random/index.html.twig\". at C:\\xampp\\htdocs\\vtnwebapp\\app\\bootstrap.php.cache:2449)"} []
[2015-02-24 10:19:01] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /admin"" at C:\xampp\htdocs\vtnwebapp\app\cache\prod\classes.php line 2420 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /admin\" at C:\\xampp\\htdocs\\vtnwebapp\\app\\cache\\prod\\classes.php:2420, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at C:\\xampp\\htdocs\\vtnwebapp\\app\\cache\\prod\\appProdUrlMatcher.php:45)"} []
[2015-02-24 10:19:16] assetic.ERROR: The template ":Security:login.html.twig" contains an error: A template that extends another one cannot have a body in ":Security:login.html.twig" at line 5. [] []
[2015-02-24 10:19:27] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /admin"" at C:\xampp\htdocs\vtnwebapp\vendor\symfony\symfony\src\Symfony\Component\HttpKernel\EventListener\RouterListener.php line 145 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /admin\" at C:\\xampp\\htdocs\\vtnwebapp\\vendor\\symfony\\symfony\\src\\Symfony\\Component\\HttpKernel\\EventListener\\RouterListener.php:145, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at C:\\xampp\\htdocs\\vtnwebapp\\app\\cache\\prod\\appProdUrlMatcher.php:58)"} []
[2015-02-24 10:24:16] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /admin"" at C:\xampp\htdocs\vtnwebapp\app\cache\prod\classes.php line 2420 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /admin\" at C:\\xampp\\htdocs\\vtnwebapp\\app\\cache\\prod\\classes.php:2420, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at C:\\xampp\\htdocs\\vtnwebapp\\app\\cache\\prod\\appProdUrlMatcher.php:58)"} []
[2015-02-24 10:29:01] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /admin"" at C:\xampp\htdocs\vtnwebapp\app\cache\prod\classes.php line 2420 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /admin\" at C:\\xampp\\htdocs\\vtnwebapp\\app\\cache\\prod\\classes.php:2420, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at C:\\xampp\\htdocs\\vtnwebapp\\app\\cache\\prod\\appProdUrlMatcher.php:58)"} []
[2015-02-24 10:29:03] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /admin"" at C:\xampp\htdocs\vtnwebapp\app\cache\prod\classes.php line 2420 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /admin\" at C:\\xampp\\htdocs\\vtnwebapp\\app\\cache\\prod\\classes.php:2420, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at C:\\xampp\\htdocs\\vtnwebapp\\app\\cache\\prod\\appProdUrlMatcher.php:58)"} []
[2015-02-24 10:29:13] assetic.ERROR: The template ":Security:login.html.twig" contains an error: A template that extends another one cannot have a body in ":Security:login.html.twig" at line 5. [] []
[2015-02-24 10:29:19] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /admin"" at C:\xampp\htdocs\vtnwebapp\vendor\symfony\symfony\src\Symfony\Component\HttpKernel\EventListener\RouterListener.php line 145 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /admin\" at C:\\xampp\\htdocs\\vtnwebapp\\vendor\\symfony\\symfony\\src\\Symfony\\Component\\HttpKernel\\EventListener\\RouterListener.php:145, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at C:\\xampp\\htdocs\\vtnwebapp\\app\\cache\\prod\\appProdUrlMatcher.php:61)"} []
[2015-02-24 10:32:57] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /admin"" at C:\xampp\htdocs\vtnwebapp\app\cache\prod\classes.php line 2420 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /admin\" at C:\\xampp\\htdocs\\vtnwebapp\\app\\cache\\prod\\classes.php:2420, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at C:\\xampp\\htdocs\\vtnwebapp\\app\\cache\\prod\\appProdUrlMatcher.php:61)"} []
[2015-02-24 10:35:31] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /app/example"" at C:\xampp\htdocs\vtnwebapp\app\cache\prod\classes.php line 2420 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /app/example\" at C:\\xampp\\htdocs\\vtnwebapp\\app\\cache\\prod\\classes.php:2420, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at C:\\xampp\\htdocs\\vtnwebapp\\app\\cache\\prod\\appProdUrlMatcher.php:61)"} []
[2015-02-24 10:35:35] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /"" at C:\xampp\htdocs\vtnwebapp\app\cache\prod\classes.php line 2420 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /\" at C:\\xampp\\htdocs\\vtnwebapp\\app\\cache\\prod\\classes.php:2420, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at C:\\xampp\\htdocs\\vtnwebapp\\app\\cache\\prod\\appProdUrlMatcher.php:61)"} []
[2015-02-24 10:37:39] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /admin"" at C:\xampp\htdocs\vtnwebapp\app\cache\prod\classes.php line 2420 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /admin\" at C:\\xampp\\htdocs\\vtnwebapp\\app\\cache\\prod\\classes.php:2420, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at C:\\xampp\\htdocs\\vtnwebapp\\app\\cache\\prod\\appProdUrlMatcher.php:61)"} []
[2015-02-24 10:38:06] assetic.ERROR: The template ":Security:login.html.twig" contains an error: A template that extends another one cannot have a body in ":Security:login.html.twig" at line 5. [] []
[2015-02-24 10:38:14] request.INFO: Matched route "login_route" (parameters: "_controller": "AppBundle\Controller\SecurityController::loginAction", "_route": "login_route") [] []
[2015-02-24 10:38:14] security.INFO: Populated SecurityContext with an anonymous Token [] []
[2015-02-24 10:38:14] request.CRITICAL: Uncaught PHP Exception ReflectionException: "Class AppBundle\Controller\Request does not exist" at C:\xampp\htdocs\vtnwebapp\app\cache\prod\classes.php line 7392 {"exception":"[object] (ReflectionException(code: 0): Class AppBundle\\Controller\\Request does not exist at C:\\xampp\\htdocs\\vtnwebapp\\app\\cache\\prod\\classes.php:7392)"} []
[2015-02-24 10:38:14] security.DEBUG: Write SecurityContext in the session [] []
[2015-02-24 10:38:19] request.INFO: Matched route "login_route" (parameters: "_controller": "AppBundle\Controller\SecurityController::loginAction", "_route": "login_route") [] []
[2015-02-24 10:38:20] security.INFO: Populated SecurityContext with an anonymous Token [] []
[2015-02-24 10:38:20] request.CRITICAL: Uncaught PHP Exception ReflectionException: "Class AppBundle\Controller\Request does not exist" at C:\xampp\htdocs\vtnwebapp\app\cache\prod\classes.php line 7392 {"exception":"[object] (ReflectionException(code: 0): Class AppBundle\\Controller\\Request does not exist at C:\\xampp\\htdocs\\vtnwebapp\\app\\cache\\prod\\classes.php:7392)"} []
[2015-02-24 10:38:20] security.DEBUG: Write SecurityContext in the session [] []
[2015-02-24 10:39:51] request.INFO: Matched route "login_route" (parameters: "_controller": "AppBundle\Controller\SecurityController::loginAction", "_route": "login_route") [] []
[2015-02-24 10:39:52] security.INFO: Populated SecurityContext with an anonymous Token [] []
[2015-02-24 10:39:52] request.CRITICAL: Uncaught PHP Exception ReflectionException: "Class AppBundle\Controller\Request does not exist" at C:\xampp\htdocs\vtnwebapp\app\cache\prod\classes.php line 7392 {"exception":"[object] (ReflectionException(code: 0): Class AppBundle\\Controller\\Request does not exist at C:\\xampp\\htdocs\\vtnwebapp\\app\\cache\\prod\\classes.php:7392)"} []
[2015-02-24 10:39:52] security.DEBUG: Write SecurityContext in the session [] []
[2015-02-24 10:40:01] assetic.ERROR: The template ":Security:login.html.twig" contains an error: A template that extends another one cannot have a body in ":Security:login.html.twig" at line 5. [] []
[2015-02-24 10:47:50] assetic.ERROR: The template ":security:login.html.twig" contains an error: A template that extends another one cannot have a body in ":security:login.html.twig" at line 5. [] []
[2015-02-24 10:47:59] request.INFO: Matched route "login_route" (parameters: "_controller": "AppBundle\Controller\SecurityController::loginAction", "_route": "login_route") [] []
[2015-02-24 10:47:59] security.INFO: Populated SecurityContext with an anonymous Token [] []
[2015-02-24 10:47:59] request.CRITICAL: Uncaught PHP Exception ReflectionException: "Class AppBundle\Controller\Request does not exist" at C:\xampp\htdocs\vtnwebapp\app\cache\prod\classes.php line 7392 {"exception":"[object] (ReflectionException(code: 0): Class AppBundle\\Controller\\Request does not exist at C:\\xampp\\htdocs\\vtnwebapp\\app\\cache\\prod\\classes.php:7392)"} []
[2015-02-24 10:47:59] security.DEBUG: Write SecurityContext in the session [] []
[2015-02-24 10:48:04] request.INFO: Matched route "login_route" (parameters: "_controller": "AppBundle\Controller\SecurityController::loginAction", "_route": "login_route") [] []
[2015-02-24 10:48:05] security.INFO: Populated SecurityContext with an anonymous Token [] []
[2015-02-24 10:48:05] request.CRITICAL: Uncaught PHP Exception ReflectionException: "Class AppBundle\Controller\Request does not exist" at C:\xampp\htdocs\vtnwebapp\app\cache\prod\classes.php line 7392 {"exception":"[object] (ReflectionException(code: 0): Class AppBundle\\Controller\\Request does not exist at C:\\xampp\\htdocs\\vtnwebapp\\app\\cache\\prod\\classes.php:7392)"} []
[2015-02-24 10:48:05] security.DEBUG: Write SecurityContext in the session [] []
[2015-02-24 10:49:28] assetic.ERROR: The template ":security:login.html.twig" contains an error: A template that extends another one cannot have a body in ":security:login.html.twig" at line 5. [] []
[2015-02-24 10:49:36] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /"" at C:\xampp\htdocs\vtnwebapp\vendor\symfony\symfony\src\Symfony\Component\HttpKernel\EventListener\RouterListener.php line 145 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /\" at C:\\xampp\\htdocs\\vtnwebapp\\vendor\\symfony\\symfony\\src\\Symfony\\Component\\HttpKernel\\EventListener\\RouterListener.php:145, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at C:\\xampp\\htdocs\\vtnwebapp\\app\\cache\\prod\\appProdUrlMatcher.php:66)"} []
[2015-02-24 10:49:41] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /"" at C:\xampp\htdocs\vtnwebapp\app\cache\prod\classes.php line 2420 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /\" at C:\\xampp\\htdocs\\vtnwebapp\\app\\cache\\prod\\classes.php:2420, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at C:\\xampp\\htdocs\\vtnwebapp\\app\\cache\\prod\\appProdUrlMatcher.php:66)"} []
[2015-02-24 10:49:46] request.INFO: Matched route "login_route" (parameters: "_controller": "AppBundle\Controller\SecurityController::loginAction", "_route": "login_route") [] []
[2015-02-24 10:49:46] security.INFO: Populated SecurityContext with an anonymous Token [] []
[2015-02-24 10:49:46] request.CRITICAL: Uncaught PHP Exception ReflectionException: "Class AppBundle\Controller\Request does not exist" at C:\xampp\htdocs\vtnwebapp\app\cache\prod\classes.php line 7392 {"exception":"[object] (ReflectionException(code: 0): Class AppBundle\\Controller\\Request does not exist at C:\\xampp\\htdocs\\vtnwebapp\\app\\cache\\prod\\classes.php:7392)"} []
[2015-02-24 10:49:46] security.DEBUG: Write SecurityContext in the session [] []
[2015-02-24 10:50:02] request.INFO: Matched route "login_route" (parameters: "_controller": "AppBundle\Controller\SecurityController::loginAction", "_route": "login_route") [] []
[2015-02-24 10:50:02] security.INFO: Populated SecurityContext with an anonymous Token [] []
[2015-02-24 10:50:02] request.CRITICAL: Uncaught PHP Exception ReflectionException: "Class AppBundle\Controller\Request does not exist" at C:\xampp\htdocs\vtnwebapp\app\cache\prod\classes.php line 7392 {"exception":"[object] (ReflectionException(code: 0): Class AppBundle\\Controller\\Request does not exist at C:\\xampp\\htdocs\\vtnwebapp\\app\\cache\\prod\\classes.php:7392)"} []
[2015-02-24 10:50:02] security.DEBUG: Write SecurityContext in the session [] []
[2015-02-24 13:19:03] request.INFO: Matched route "login_route" (parameters: "_controller": "AppBundle\Controller\SecurityController::loginAction", "_route": "login_route") [] []
[2015-02-24 13:19:03] security.INFO: Populated SecurityContext with an anonymous Token [] []
[2015-02-24 13:19:04] request.CRITICAL: Uncaught PHP Exception ReflectionException: "Class AppBundle\Controller\Request does not exist" at C:\xampp\htdocs\vtnwebapp\app\cache\prod\classes.php line 7392 {"exception":"[object] (ReflectionException(code: 0): Class AppBundle\\Controller\\Request does not exist at C:\\xampp\\htdocs\\vtnwebapp\\app\\cache\\prod\\classes.php:7392)"} []
[2015-02-24 13:19:05] security.DEBUG: Write SecurityContext in the session [] []
[2015-02-24 13:19:17] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /"" at C:\xampp\htdocs\vtnwebapp\app\cache\prod\classes.php line 2420 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /\" at C:\\xampp\\htdocs\\vtnwebapp\\app\\cache\\prod\\classes.php:2420, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at C:\\xampp\\htdocs\\vtnwebapp\\app\\cache\\prod\\appProdUrlMatcher.php:66)"} []
[2015-02-24 13:19:21] request.INFO: Matched route "login_route" (parameters: "_controller": "AppBundle\Controller\SecurityController::loginAction", "_route": "login_route") [] []
[2015-02-24 13:19:21] security.INFO: Populated SecurityContext with an anonymous Token [] []
[2015-02-24 13:19:21] request.CRITICAL: Uncaught PHP Exception ReflectionException: "Class AppBundle\Controller\Request does not exist" at C:\xampp\htdocs\vtnwebapp\app\cache\prod\classes.php line 7392 {"exception":"[object] (ReflectionException(code: 0): Class AppBundle\\Controller\\Request does not exist at C:\\xampp\\htdocs\\vtnwebapp\\app\\cache\\prod\\classes.php:7392)"} []
[2015-02-24 13:19:21] security.DEBUG: Write SecurityContext in the session [] []
[2015-02-24 13:33:41] assetic.ERROR: The template ":security:login.html.twig" contains an error: A template that extends another one cannot have a body in ":security:login.html.twig" at line 5. [] []
[2015-02-24 14:37:36] php.EMERGENCY: Fatal Parse Error: syntax error, unexpected '...' (T_ELLIPSIS) {"type":1,"file":"C:\\xampp\\htdocs\\vtnwebapp\\src\\WebserviceUserBundle\\Security\\User\\WebserviceUserProvider.php","line":16,"level":22527,"stack":[]} []
[2015-02-24 15:14:20] php.EMERGENCY: Fatal Parse Error: syntax error, unexpected '...' (T_ELLIPSIS) {"type":1,"file":"C:\\xampp\\htdocs\\vtnwebapp\\src\\WebserviceUserBundle\\Security\\User\\WebserviceUserProvider.php","line":16,"level":22527,"stack":[]} []
[2015-02-24 15:16:24] assetic.ERROR: The template ":security:login.html.twig" contains an error: A template that extends another one cannot have a body in ":security:login.html.twig" at line 5. [] []
[2015-02-24 15:16:58] request.INFO: Matched route "login_route" (parameters: "_controller": "AppBundle\Controller\SecurityController::loginAction", "_route": "login_route") [] []
[2015-02-24 15:16:58] security.INFO: Populated SecurityContext with an anonymous Token [] []
[2015-02-24 15:16:58] request.CRITICAL: Uncaught PHP Exception ReflectionException: "Class AppBundle\Controller\Request does not exist" at C:\xampp\htdocs\vtnwebapp\app\cache\prod\classes.php line 7392 {"exception":"[object] (ReflectionException(code: 0): Class AppBundle\\Controller\\Request does not exist at C:\\xampp\\htdocs\\vtnwebapp\\app\\cache\\prod\\classes.php:7392)"} []
[2015-02-24 15:16:58] security.DEBUG: Write SecurityContext in the session [] []
[2015-02-24 15:17:03] request.INFO: Matched route "login_route" (parameters: "_controller": "AppBundle\Controller\SecurityController::loginAction", "_route": "login_route") [] []
[2015-02-24 15:17:03] security.INFO: Populated SecurityContext with an anonymous Token [] []
[2015-02-24 15:17:03] request.CRITICAL: Uncaught PHP Exception ReflectionException: "Class AppBundle\Controller\Request does not exist" at C:\xampp\htdocs\vtnwebapp\app\cache\prod\classes.php line 7392 {"exception":"[object] (ReflectionException(code: 0): Class AppBundle\\Controller\\Request does not exist at C:\\xampp\\htdocs\\vtnwebapp\\app\\cache\\prod\\classes.php:7392)"} []
[2015-02-24 15:17:03] security.DEBUG: Write SecurityContext in the session [] []
[2015-02-24 15:17:54] request.INFO: Matched route "login_route" (parameters: "_controller": "AppBundle\Controller\SecurityController::loginAction", "_route": "login_route") [] []
[2015-02-24 15:17:55] security.INFO: Populated SecurityContext with an anonymous Token [] []
[2015-02-24 15:17:55] request.CRITICAL: Uncaught PHP Exception ReflectionException: "Class AppBundle\Controller\Request does not exist" at C:\xampp\htdocs\vtnwebapp\app\cache\prod\classes.php line 7392 {"exception":"[object] (ReflectionException(code: 0): Class AppBundle\\Controller\\Request does not exist at C:\\xampp\\htdocs\\vtnwebapp\\app\\cache\\prod\\classes.php:7392)"} []
[2015-02-24 15:17:55] security.DEBUG: Write SecurityContext in the session [] []
[2015-02-24 15:23:17] request.INFO: Matched route "login_route" (parameters: "_controller": "AppBundle\Controller\SecurityController::loginAction", "_route": "login_route") [] []
[2015-02-24 15:23:17] security.INFO: Populated SecurityContext with an anonymous Token [] []
[2015-02-24 15:23:17] request.CRITICAL: Uncaught PHP Exception ReflectionException: "Class AppBundle\Controller\Request does not exist" at C:\xampp\htdocs\vtnwebapp\app\cache\prod\classes.php line 7392 {"exception":"[object] (ReflectionException(code: 0): Class AppBundle\\Controller\\Request does not exist at C:\\xampp\\htdocs\\vtnwebapp\\app\\cache\\prod\\classes.php:7392)"} []
[2015-02-24 15:23:17] security.DEBUG: Write SecurityContext in the session [] []
[2015-02-24 15:24:27] assetic.ERROR: The template ":security:login.html.twig" contains an error: A template that extends another one cannot have a body in ":security:login.html.twig" at line 5. [] []
[2015-02-24 15:24:43] request.INFO: Matched route "login_route" (parameters: "_controller": "AppBundle\Controller\SecurityController::loginAction", "_route": "login_route") [] []
[2015-02-24 15:24:43] security.INFO: Populated SecurityContext with an anonymous Token [] []
[2015-02-24 15:24:43] request.CRITICAL: Uncaught PHP Exception ReflectionException: "Class AppBundle\Controller\Request does not exist" at C:\xampp\htdocs\vtnwebapp\app\cache\prod\classes.php line 7392 {"exception":"[object] (ReflectionException(code: 0): Class AppBundle\\Controller\\Request does not exist at C:\\xampp\\htdocs\\vtnwebapp\\app\\cache\\prod\\classes.php:7392)"} []
[2015-02-24 15:24:43] security.DEBUG: Write SecurityContext in the session [] []
[2015-02-24 15:41:03] assetic.ERROR: The template ":security:login.html.twig" contains an error: A template that extends another one cannot have a body in ":security:login.html.twig" at line 5. [] []
[2015-03-01 11:08:29] php.EMERGENCY: Fatal Compile Error: Declaration of Symfony\Component\Security\Core\User\InMemoryUserProvider::loadUserByUsername() must be compatible with Symfony\Component\Security\Core\User\UserProviderInterface::loadUserByUsername($username, $password) {"type":1,"file":"C:\\xampp\\htdocs\\vtnwebapp\\vendor\\symfony\\symfony\\src\\Symfony\\Component\\Security\\Core\\User\\InMemoryUserProvider.php","line":26,"level":22527,"stack":[]} []
[2015-03-01 11:35:56] request.INFO: Matched route "app_default_admin" (parameters: "_controller": "AppBundle\Controller\DefaultController::adminAction", "_route": "app_default_admin") [] []
[2015-03-01 11:35:56] php.EMERGENCY: Fatal Compile Error: Declaration of Symfony\Component\Security\Core\User\InMemoryUserProvider::loadUserByUsername() must be compatible with Symfony\Component\Security\Core\User\UserProviderInterface::loadUserByUsername($username, $password) {"type":1,"file":"C:\\xampp\\htdocs\\vtnwebapp\\vendor\\symfony\\symfony\\src\\Symfony\\Component\\Security\\Core\\User\\InMemoryUserProvider.php","line":26,"level":22527,"stack":[]} []
[2015-03-01 11:35:56] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\FatalErrorException: "Compile Error: Declaration of Symfony\Component\Security\Core\User\InMemoryUserProvider::loadUserByUsername() must be compatible with Symfony\Component\Security\Core\User\UserProviderInterface::loadUserByUsername($username, $password)" at C:\xampp\htdocs\vtnwebapp\vendor\symfony\symfony\src\Symfony\Component\Security\Core\User\InMemoryUserProvider.php line 26 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalErrorException(code: 0): Compile Error: Declaration of Symfony\\Component\\Security\\Core\\User\\InMemoryUserProvider::loadUserByUsername() must be compatible with Symfony\\Component\\Security\\Core\\User\\UserProviderInterface::loadUserByUsername($username, $password) at C:\\xampp\\htdocs\\vtnwebapp\\vendor\\symfony\\symfony\\src\\Symfony\\Component\\Security\\Core\\User\\InMemoryUserProvider.php:26)"} []
[2015-03-03 08:00:48] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /random"" at C:\xampp\htdocs\vtnwebapp\vendor\symfony\symfony\src\Symfony\Component\HttpKernel\EventListener\RouterListener.php line 145 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /random\" at C:\\xampp\\htdocs\\vtnwebapp\\vendor\\symfony\\symfony\\src\\Symfony\\Component\\HttpKernel\\EventListener\\RouterListener.php:145, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at C:\\xampp\\htdocs\\vtnwebapp\\app\\cache\\prod\\appProdUrlMatcher.php:71)"} []
[2015-03-03 08:19:14] request.INFO: Matched route "login_route" (parameters: "_controller": "AppBundle\Controller\SecurityController::loginAction", "_route": "login_route") [] []
[2015-03-03 08:19:14] security.INFO: Populated SecurityContext with an anonymous Token [] []
[2015-03-03 08:19:15] request.CRITICAL: Uncaught PHP Exception Twig_Error_Syntax: "A template that extends another one cannot have a body in "AppBundle::login.html.twig" at line 4." at C:\xampp\htdocs\vtnwebapp\vendor\twig\twig\lib\Twig\Parser.php line 370 {"exception":"[object] (Twig_Error_Syntax(code: 0): A template that extends another one cannot have a body in \"AppBundle::login.html.twig\" at line 4. at C:\\xampp\\htdocs\\vtnwebapp\\vendor\\twig\\twig\\lib\\Twig\\Parser.php:370)"} []
[2015-03-03 08:19:15] security.DEBUG: Write SecurityContext in the session [] []
[2015-03-03 08:19:58] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /adin"" at C:\xampp\htdocs\vtnwebapp\app\cache\prod\classes.php line 2420 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /adin\" at C:\\xampp\\htdocs\\vtnwebapp\\app\\cache\\prod\\classes.php:2420, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at C:\\xampp\\htdocs\\vtnwebapp\\app\\cache\\prod\\appProdUrlMatcher.php:71)"} []
[2015-03-03 08:20:01] request.INFO: Matched route "login_route" (parameters: "_controller": "AppBundle\Controller\SecurityController::loginAction", "_route": "login_route") [] []
[2015-03-03 08:20:01] security.INFO: Populated SecurityContext with an anonymous Token [] []
[2015-03-03 08:20:01] request.CRITICAL: Uncaught PHP Exception Twig_Error_Syntax: "A template that extends another one cannot have a body in "AppBundle::login.html.twig" at line 4." at C:\xampp\htdocs\vtnwebapp\vendor\twig\twig\lib\Twig\Parser.php line 370 {"exception":"[object] (Twig_Error_Syntax(code: 0): A template that extends another one cannot have a body in \"AppBundle::login.html.twig\" at line 4. at C:\\xampp\\htdocs\\vtnwebapp\\vendor\\twig\\twig\\lib\\Twig\\Parser.php:370)"} []
[2015-03-03 08:20:01] security.DEBUG: Write SecurityContext in the session [] []
[2015-03-03 08:25:00] request.INFO: Matched route "homepage" (parameters: "_controller": "AppBundle\Controller\DefaultController::indexAction", "_route": "homepage") [] []
[2015-03-03 08:25:00] security.INFO: Populated SecurityContext with an anonymous Token [] []
[2015-03-03 08:25:00] request.CRITICAL: Uncaught PHP Exception Twig_Error_Runtime: "An exception has been thrown during the rendering of a template ("Catchable Fatal Error: Argument 1 passed to Symfony\Component\Form\FormRenderer::renderBlock() must be an instance of Symfony\Component\Form\FormView, null given, called in C:\xampp\htdocs\vtnwebapp\app\cache\prod\twig\f3\cd\c9c5d8083d320b3a128931681c6e76f33e2380a5a31b0e4b2fc19484dbb5.php on line 55 and defined") in "AppBundle::layout.html.twig" at line 23." at C:\xampp\htdocs\vtnwebapp\vendor\twig\twig\lib\Twig\Template.php line 304 {"exception":"[object] (Twig_Error_Runtime(code: 0): An exception has been thrown during the rendering of a template (\"Catchable Fatal Error: Argument 1 passed to Symfony\\Component\\Form\\FormRenderer::renderBlock() must be an instance of Symfony\\Component\\Form\\FormView, null given, called in C:\\xampp\\htdocs\\vtnwebapp\\app\\cache\\prod\\twig\\f3\\cd\\c9c5d8083d320b3a128931681c6e76f33e2380a5a31b0e4b2fc19484dbb5.php on line 55 and defined\") in \"AppBundle::layout.html.twig\" at line 23. at C:\\xampp\\htdocs\\vtnwebapp\\vendor\\twig\\twig\\lib\\Twig\\Template.php:304, Symfony\\Component\\Debug\\Exception\\ContextErrorException(code: 0): Catchable Fatal Error: Argument 1 passed to Symfony\\Component\\Form\\FormRenderer::renderBlock() must be an instance of Symfony\\Component\\Form\\FormView, null given, called in C:\\xampp\\htdocs\\vtnwebapp\\app\\cache\\prod\\twig\\f3\\cd\\c9c5d8083d320b3a128931681c6e76f33e2380a5a31b0e4b2fc19484dbb5.php on line 55 and defined at C:\\xampp\\htdocs\\vtnwebapp\\vendor\\symfony\\symfony\\src\\Symfony\\Component\\Form\\FormRenderer.php:106)"} []
[2015-03-03 08:25:00] security.DEBUG: Write SecurityContext in the session [] []
[2015-03-03 08:26:20] request.INFO: Matched route "homepage" (parameters: "_controller": "AppBundle\Controller\DefaultController::indexAction", "_route": "homepage") [] []
[2015-03-03 08:26:20] security.INFO: Populated SecurityContext with an anonymous Token [] []
[2015-03-03 08:26:20] request.CRITICAL: Uncaught PHP Exception Twig_Error_Runtime: "An exception has been thrown during the rendering of a template ("Catchable Fatal Error: Argument 1 passed to Symfony\Component\Form\FormRenderer::searchAndRenderBlock() must be an instance of Symfony\Component\Form\FormView, null given, called in C:\xampp\htdocs\vtnwebapp\app\cache\prod\twig\f3\cd\c9c5d8083d320b3a128931681c6e76f33e2380a5a31b0e4b2fc19484dbb5.php on line 64 and defined") in "AppBundle::layout.html.twig" at line 26." at C:\xampp\htdocs\vtnwebapp\vendor\twig\twig\lib\Twig\Template.php line 304 {"exception":"[object] (Twig_Error_Runtime(code: 0): An exception has been thrown during the rendering of a template (\"Catchable Fatal Error: Argument 1 passed to Symfony\\Component\\Form\\FormRenderer::searchAndRenderBlock() must be an instance of Symfony\\Component\\Form\\FormView, null given, called in C:\\xampp\\htdocs\\vtnwebapp\\app\\cache\\prod\\twig\\f3\\cd\\c9c5d8083d320b3a128931681c6e76f33e2380a5a31b0e4b2fc19484dbb5.php on line 64 and defined\") in \"AppBundle::layout.html.twig\" at line 26. at C:\\xampp\\htdocs\\vtnwebapp\\vendor\\twig\\twig\\lib\\Twig\\Template.php:304, Symfony\\Component\\Debug\\Exception\\ContextErrorException(code: 0): Catchable Fatal Error: Argument 1 passed to Symfony\\Component\\Form\\FormRenderer::searchAndRenderBlock() must be an instance of Symfony\\Component\\Form\\FormView, null given, called in C:\\xampp\\htdocs\\vtnwebapp\\app\\cache\\prod\\twig\\f3\\cd\\c9c5d8083d320b3a128931681c6e76f33e2380a5a31b0e4b2fc19484dbb5.php on line 64 and defined at C:\\xampp\\htdocs\\vtnwebapp\\vendor\\symfony\\symfony\\src\\Symfony\\Component\\Form\\FormRenderer.php:165)"} []
[2015-03-03 08:26:20] security.DEBUG: Write SecurityContext in the session [] []
[2015-03-03 08:36:08] request.INFO: Matched route "homepage" (parameters: "_controller": "AppBundle\Controller\DefaultController::indexAction", "_route": "homepage") [] []
[2015-03-03 08:36:08] security.INFO: Populated SecurityContext with an anonymous Token [] []
[2015-03-03 08:36:08] request.CRITICAL: Uncaught PHP Exception Twig_Error_Runtime: "The merge filter only works with arrays or hashes; NULL and array given in "AppBundle::layout.html.twig" at line 74." at C:\xampp\htdocs\vtnwebapp\vendor\twig\twig\lib\Twig\Extension\Core.php line 686 {"exception":"[object] (Twig_Error_Runtime(code: 0): The merge filter only works with arrays or hashes; NULL and array given in \"AppBundle::layout.html.twig\" at line 74. at C:\\xampp\\htdocs\\vtnwebapp\\vendor\\twig\\twig\\lib\\Twig\\Extension\\Core.php:686)"} []
[2015-03-03 08:36:08] security.DEBUG: Write SecurityContext in the session [] []
[2015-03-03 08:38:36] request.INFO: Matched route "homepage" (parameters: "_controller": "AppBundle\Controller\DefaultController::indexAction", "_route": "homepage") [] []
[2015-03-03 08:38:36] security.INFO: Populated SecurityContext with an anonymous Token [] []
[2015-03-03 08:38:36] request.CRITICAL: Uncaught PHP Exception Twig_Error_Runtime: "The merge filter only works with arrays or hashes; NULL and array given in "AppBundle::layout.html.twig" at line 74." at C:\xampp\htdocs\vtnwebapp\vendor\twig\twig\lib\Twig\Extension\Core.php line 686 {"exception":"[object] (Twig_Error_Runtime(code: 0): The merge filter only works with arrays or hashes; NULL and array given in \"AppBundle::layout.html.twig\" at line 74. at C:\\xampp\\htdocs\\vtnwebapp\\vendor\\twig\\twig\\lib\\Twig\\Extension\\Core.php:686)"} []
[2015-03-03 08:38:36] security.DEBUG: Write SecurityContext in the session [] []
[2015-03-03 08:41:31] request.INFO: Matched route "homepage" (parameters: "_controller": "AppBundle\Controller\DefaultController::indexAction", "_route": "homepage") [] []
[2015-03-03 08:41:31] security.INFO: Populated SecurityContext with an anonymous Token [] []
[2015-03-03 08:41:31] request.CRITICAL: Uncaught PHP Exception Twig_Error_Runtime: "An exception has been thrown during the rendering of a template ("Catchable Fatal Error: Argument 1 passed to Symfony\Component\Form\FormRenderer::searchAndRenderBlock() must be an instance of Symfony\Component\Form\FormView, null given, called in C:\xampp\htdocs\vtnwebapp\app\cache\prod\twig\f3\cd\c9c5d8083d320b3a128931681c6e76f33e2380a5a31b0e4b2fc19484dbb5.php on line 61 and defined") in "AppBundle::layout.html.twig" at line 26." at C:\xampp\htdocs\vtnwebapp\vendor\twig\twig\lib\Twig\Template.php line 304 {"exception":"[object] (Twig_Error_Runtime(code: 0): An exception has been thrown during the rendering of a template (\"Catchable Fatal Error: Argument 1 passed to Symfony\\Component\\Form\\FormRenderer::searchAndRenderBlock() must be an instance of Symfony\\Component\\Form\\FormView, null given, called in C:\\xampp\\htdocs\\vtnwebapp\\app\\cache\\prod\\twig\\f3\\cd\\c9c5d8083d320b3a128931681c6e76f33e2380a5a31b0e4b2fc19484dbb5.php on line 61 and defined\") in \"AppBundle::layout.html.twig\" at line 26. at C:\\xampp\\htdocs\\vtnwebapp\\vendor\\twig\\twig\\lib\\Twig\\Template.php:304, Symfony\\Component\\Debug\\Exception\\ContextErrorException(code: 0): Catchable Fatal Error: Argument 1 passed to Symfony\\Component\\Form\\FormRenderer::searchAndRenderBlock() must be an instance of Symfony\\Component\\Form\\FormView, null given, called in C:\\xampp\\htdocs\\vtnwebapp\\app\\cache\\prod\\twig\\f3\\cd\\c9c5d8083d320b3a128931681c6e76f33e2380a5a31b0e4b2fc19484dbb5.php on line 61 and defined at C:\\xampp\\htdocs\\vtnwebapp\\vendor\\symfony\\symfony\\src\\Symfony\\Component\\Form\\FormRenderer.php:165)"} []
[2015-03-03 08:41:31] security.DEBUG: Write SecurityContext in the session [] []
[2015-03-03 15:27:35] request.INFO: Matched route "login_route" (parameters: "_controller": "AppBundle\Controller\SecurityController::loginAction", "_route": "login_route") [] []
[2015-03-03 15:27:35] security.INFO: Populated SecurityContext with an anonymous Token [] []
[2015-03-03 15:27:36] request.CRITICAL: Uncaught PHP Exception Twig_Error_Runtime: "An exception has been thrown during the rendering of a template ("Catchable Fatal Error: Argument 1 passed to Symfony\Component\Form\FormRenderer::searchAndRenderBlock() must be an instance of Symfony\Component\Form\FormView, null given, called in C:\xampp\htdocs\vtnwebapp\app\cache\prod\twig\0b\9c\03ac374a5cf8fe45b39cef0db806df68963fdd20708a529466e424b13e3c.php on line 54 and defined") in "AppBundle::login.html.twig" at line 8." at C:\xampp\htdocs\vtnwebapp\app\cache\prod\classes.php line 5113 {"exception":"[object] (Twig_Error_Runtime(code: 0): An exception has been thrown during the rendering of a template (\"Catchable Fatal Error: Argument 1 passed to Symfony\\Component\\Form\\FormRenderer::searchAndRenderBlock() must be an instance of Symfony\\Component\\Form\\FormView, null given, called in C:\\xampp\\htdocs\\vtnwebapp\\app\\cache\\prod\\twig\\0b\\9c\\03ac374a5cf8fe45b39cef0db806df68963fdd20708a529466e424b13e3c.php on line 54 and defined\") in \"AppBundle::login.html.twig\" at line 8. at C:\\xampp\\htdocs\\vtnwebapp\\app\\cache\\prod\\classes.php:5113, Symfony\\Component\\Debug\\Exception\\ContextErrorException(code: 0): Catchable Fatal Error: Argument 1 passed to Symfony\\Component\\Form\\FormRenderer::searchAndRenderBlock() must be an instance of Symfony\\Component\\Form\\FormView, null given, called in C:\\xampp\\htdocs\\vtnwebapp\\app\\cache\\prod\\twig\\0b\\9c\\03ac374a5cf8fe45b39cef0db806df68963fdd20708a529466e424b13e3c.php on line 54 and defined at C:\\xampp\\htdocs\\vtnwebapp\\vendor\\symfony\\symfony\\src\\Symfony\\Component\\Form\\FormRenderer.php:165)"} []
[2015-03-03 15:27:37] security.DEBUG: Write SecurityContext in the session [] []
[2015-03-08 14:28:44] request.INFO: Matched route "admin" (parameters: "_controller": "AppBundle\Controller\DefaultController::adminAction", "_route": "admin") [] []
[2015-03-08 14:28:44] security.INFO: Populated SecurityContext with an anonymous Token [] []
[2015-03-08 14:28:45] request.CRITICAL: Uncaught PHP Exception Twig_Error_Syntax: "The function "dump" does not exist in "AppBundle::admin.html.twig" at line 48" at C:\xampp\htdocs\vtnwebapp\vendor\twig\twig\lib\Twig\ExpressionParser.php line 568 {"exception":"[object] (Twig_Error_Syntax(code: 0): The function \"dump\" does not exist in \"AppBundle::admin.html.twig\" at line 48 at C:\\xampp\\htdocs\\vtnwebapp\\vendor\\twig\\twig\\lib\\Twig\\ExpressionParser.php:568)"} []
[2015-03-08 14:28:45] security.DEBUG: Write SecurityContext in the session [] []
[2015-03-08 14:29:00] request.INFO: Matched route "admin" (parameters: "_controller": "AppBundle\Controller\DefaultController::adminAction", "_route": "admin") [] []
[2015-03-08 14:29:00] security.INFO: Populated SecurityContext with an anonymous Token [] []
[2015-03-08 14:29:00] request.CRITICAL: Uncaught PHP Exception Twig_Error_Syntax: "The function "dump" does not exist in "AppBundle::admin.html.twig" at line 48" at C:\xampp\htdocs\vtnwebapp\vendor\twig\twig\lib\Twig\ExpressionParser.php line 568 {"exception":"[object] (Twig_Error_Syntax(code: 0): The function \"dump\" does not exist in \"AppBundle::admin.html.twig\" at line 48 at C:\\xampp\\htdocs\\vtnwebapp\\vendor\\twig\\twig\\lib\\Twig\\ExpressionParser.php:568)"} []
[2015-03-08 14:29:01] security.DEBUG: Write SecurityContext in the session [] []
[2015-03-08 14:29:39] request.INFO: Matched route "admin" (parameters: "_controller": "AppBundle\Controller\DefaultController::adminAction", "_route": "admin") [] []
[2015-03-08 14:29:39] security.INFO: Populated SecurityContext with an anonymous Token [] []
[2015-03-08 14:29:40] request.CRITICAL: Uncaught PHP Exception Twig_Error_Syntax: "The function "dump" does not exist in "AppBundle::admin.html.twig" at line 48" at C:\xampp\htdocs\vtnwebapp\vendor\twig\twig\lib\Twig\ExpressionParser.php line 568 {"exception":"[object] (Twig_Error_Syntax(code: 0): The function \"dump\" does not exist in \"AppBundle::admin.html.twig\" at line 48 at C:\\xampp\\htdocs\\vtnwebapp\\vendor\\twig\\twig\\lib\\Twig\\ExpressionParser.php:568)"} []
[2015-03-08 14:29:40] security.DEBUG: Write SecurityContext in the session [] []
[2015-03-08 14:31:03] request.INFO: Matched route "admin" (parameters: "_controller": "AppBundle\Controller\DefaultController::adminAction", "_route": "admin") [] []
[2015-03-08 14:31:03] security.INFO: Populated SecurityContext with an anonymous Token [] []
[2015-03-08 14:31:04] request.CRITICAL: Uncaught PHP Exception Twig_Error_Syntax: "The function "dump" does not exist in "AppBundle::admin.html.twig" at line 48" at C:\xampp\htdocs\vtnwebapp\vendor\twig\twig\lib\Twig\ExpressionParser.php line 568 {"exception":"[object] (Twig_Error_Syntax(code: 0): The function \"dump\" does not exist in \"AppBundle::admin.html.twig\" at line 48 at C:\\xampp\\htdocs\\vtnwebapp\\vendor\\twig\\twig\\lib\\Twig\\ExpressionParser.php:568)"} []
[2015-03-08 14:31:04] security.DEBUG: Write SecurityContext in the session [] []
[2015-03-08 14:31:29] request.INFO: Matched route "admin" (parameters: "_controller": "AppBundle\Controller\DefaultController::adminAction", "_route": "admin") [] []
[2015-03-08 14:31:29] security.INFO: Populated SecurityContext with an anonymous Token [] []
[2015-03-08 14:31:30] request.CRITICAL: Uncaught PHP Exception Twig_Error_Syntax: "The function "dump" does not exist in "AppBundle::admin.html.twig" at line 48" at C:\xampp\htdocs\vtnwebapp\vendor\twig\twig\lib\Twig\ExpressionParser.php line 568 {"exception":"[object] (Twig_Error_Syntax(code: 0): The function \"dump\" does not exist in \"AppBundle::admin.html.twig\" at line 48 at C:\\xampp\\htdocs\\vtnwebapp\\vendor\\twig\\twig\\lib\\Twig\\ExpressionParser.php:568)"} []
[2015-03-08 14:31:30] security.DEBUG: Write SecurityContext in the session [] []
[2015-03-11 05:00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /admin_dev.php/admin"" at C:\xampp\htdocs\vtnwebapp\app\cache\prod\classes.php line 2420 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /admin_dev.php/admin\" at C:\\xampp\\htdocs\\vtnwebapp\\app\\cache\\prod\\classes.php:2420, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at C:\\xampp\\htdocs\\vtnwebapp\\app\\cache\\prod\\appProdUrlMatcher.php:71)"} []
[2015-03-11 05:00:15] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /admin_dev.php/admin"" at C:\xampp\htdocs\vtnwebapp\app\cache\prod\classes.php line 2420 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /admin_dev.php/admin\" at C:\\xampp\\htdocs\\vtnwebapp\\app\\cache\\prod\\classes.php:2420, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at C:\\xampp\\htdocs\\vtnwebapp\\app\\cache\\prod\\appProdUrlMatcher.php:71)"} []
[2015-03-11 05:01:05] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /admin_dev.php/admin"" at C:\xampp\htdocs\vtnwebapp\app\cache\prod\classes.php line 2420 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /admin_dev.php/admin\" at C:\\xampp\\htdocs\\vtnwebapp\\app\\cache\\prod\\classes.php:2420, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at C:\\xampp\\htdocs\\vtnwebapp\\app\\cache\\prod\\appProdUrlMatcher.php:71)"} []
